src/pages/AppDetail.svelte

```diff
@@ -5,19 +5,46 @@
   import FilterInput from "../components/FilterInput.svelte";
   import AppAlert from "../components/AppAlert.svelte";
   import NotFound from "../components/NotFound.svelte";
+  import Pagination, {
+    makePages,
+    goToPage,
+  } from "../components/Pagination.svelte";
 
   export let params;
   const URL = `data/${params.app}/index.json`;
   let app;
+  let to;
+  let from;
+  let total;
+  let metrics = [];
+  let pages = [];
+  let paginationData;
```

Comment:
I think `paginationState` might be a slightly better variable name, since "data" is quite ambiguous.

```suggestion
  let paginationState;
```

Reply:
Excellent!